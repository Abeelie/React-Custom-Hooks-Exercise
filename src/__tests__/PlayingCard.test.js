import React from "react";
import { render } from "@testing-library/react";
import PlayingCard from "../Components/playing-card/PlayingCard";


it("renders without crashing", function() {
  render(<PlayingCard />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<PlayingCard />);
  expect(asFragment()).toMatchSnapshot();
});