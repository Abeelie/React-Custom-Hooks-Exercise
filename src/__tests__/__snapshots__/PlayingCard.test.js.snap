// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <img
    alt="playing card"
    class="PlayingCard Card"
  />
</DocumentFragment>
`;
